tests/reference/ast_new-string1-96b90b3.stdout

```diff
@@ -129,7 +129,7 @@
     (Expr
         (ConstantStr
             "Text"
-            ()
+            "u"
```

Comment:
This change fixes a bug?

Reply:
It seems that the prefixes "u" and "U" were meaningful for Python 2. In Python 3, all string literals are Unicode by default. Thus, it seems the prefixes "u" and "U" do not add much/any value in Python 3 (Reference https://chat.openai.com/share/1270fc0a-fb93-4638-822d-3d1619488027).

With respect to the above, I think it does not as such fix a bug. Previously, we were passing the kind only for "u" (prefix). Currently, we also pass the kind value for "U" (capital `U`) (prefix).